recipes/vscode-jupyter/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "vscode-jupyter" %}
+{% set version = "2020.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 99502db43f9f2ac4185d55ae51a70c81ffe9353983dcaca5cd1a7cf29f783a11
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - nodejs 12.14.1
```

Can the pin be less strict?
```suggestion
    - nodejs 12
```

---

In the [prerequisites](https://github.com/microsoft/vscode-jupyter/blob/main/CONTRIBUTING.md#prerequisites) they explicitly mention nodejs 12.14.1. I felt like sticking to the instructions in the source repository is a good idea, but I'm totally open to making this pin less strict, if that doesn't break the build.

---

Made the node pinning less strict in https://github.com/conda-forge/staged-recipes/pull/14117/commits/395466c37c17d5642374d56a5a0c2b01f736f9d7 and the [CI pipeline still succeeds](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=283616&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd)